app/src/main/java/mobile/programming/musicApp/ui/main/MainViewModel.kt

```diff
@@ -1,4 +1,4 @@
-package mobile.programming.musicapp.ui.main
+package mobile.programming.musicApp.ui.main
```

Comment:
Here, I have also changed the multi-name package name to camelcase.

Reply:
Good noting, I did not think about correct naming conventions when renaming the project for Firebase